glidein_start.sh

```diff
@@ -198,8 +198,23 @@ export PATH=$PATH:$_condor_SBIN:$PWD/glideinExec/bin
 export LD_LIBRARY_PATH=$_condor_LIB:$_condor_LIB/condor:$LD_LIBRARY_PATH
 
 # run condor
-exec glideinExec/sbin/condor_master -dyn -f -r 1200
-
-# clean up after ourselves
-cd ..
-rm -rf glidein
+trap 'kill -TERM $PID; if [ -n $PID_LOG_SHIPPER ]; then kill -TERM $PID_LOG_SHIPPER; fi' SIGTERM SIGKILL
```

Comment:
Note that you can't actually trap `SIGKILL`.  That signal is handled at the OS level, and never appears to your process.  Your process just dies.

Reply:
One thing you could do is trap `SIGINT` (control-c).  That might be useful when running as a user, or if any batch system actually uses it.

Reply:
@dsschult I changed sigkill to sigint